src/backend/TrafficCourtsApi/DisputeApi.Web/Features/Tickets/ticketSearch.openapi.json

```diff
@@ -0,0 +1,162 @@
+﻿{
```

Comment:
I would hope our OpenAPI specs would be better documented. I assume this is generated, and the reason there are no docs is because we do not have any docs on the poco objects, etc.

Reply:
comment added and better doc generated.